client/state/selectors/is-jetpack-remote-install-complete.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns success status of an attempted remote install
+ * of the jetpack plugin to the .org site at the given url.
+ *
+ * @param {Object} state - Global state tree
+ * @param {String} url - .org site URL
+ * @return {bool} - True if installation and activation was successful
```

Comment:
I don't believe the `-` belongs in these docs. I believe this should be long form `boolean`.